docs/geedocs/5.2.3/answer/6051700.html

```diff
@@ -126,6 +126,27 @@ <h3>Symbolic links</h3>
   <li><strong>soft-link</strong>: the disconnected database folder and published_dbs folder are on different logical volumes.</li>
   <li>If the disconnected database folder is in <code>/tmp</code>, the directory files will be copied. </li>
 </ul>
+<h3>Rewrite KML Layer URLs</h3>
+<p>If the disconnected database contains KML Layers, the KML Layer's URLs may require adjustments in order to resolve the KML content in a disconnected environment.   (See <a href="../answer/2760457.html">Add KML Layers </a>). Use the <code>gerewritedbroot</code> command to update the disconnected database KML Links with new hostname, ports paths or filenames. </p>
```

Comment:
Minor:

* Remove space between 'Add KML Layers' and the closing bracket.
* Add a comma after 'ports'.